packages/jest-transform/src/types.ts

```diff
@@ -70,5 +72,6 @@ export interface Transformer<OptionType = unknown> {
     sourceText: string,
     sourcePath: Config.Path,
     options: TransformOptions,
+    cacheFS?: StringMap,
```

Comment:
this should be mandatory arg but then this https://github.com/facebook/jest/blob/master/packages/jest-repl/src/cli/repl.ts#L28 failed. Im not familiar with `jest-repl` so Idk if I should hardcode `new Map()` there